README.md

```diff
@@ -17,27 +17,27 @@ You can find these dashboards in:
 
 <table>
   <thead>
-    <tr>
-      <th width="50%">Entrance 1: GitHub User's Profile Page</th>
-      <th width="50%">Entrance 2: GitHub Repository Page</th>
-    </tr>
+	<tr>
+	  <th width="50%">Entrance 1: GitHub User's Profile Page</th>
+	  <th width="50%">Entrance 2: GitHub Repository Page</th>
+	</tr>
   </thead>
   <tbody>
-    <tr>
-      <td>
-        <img
-          src="https://user-images.githubusercontent.com/32434520/180445548-63d5e0ce-635f-4e7b-bed7-e4bcbf2dc8c4.png"
-        />
-      </td>
-      <td>
-        <img
-          src="https://user-images.githubusercontent.com/32434520/180447103-76ff1e25-ec35-4e7f-bd54-9d98545ca1df.png"
-        />
-        <img
-          src="https://user-images.githubusercontent.com/32434520/180446790-50b6a53b-119e-4b74-a08d-dda146fb9f29.png"
-        />
-      </td>
-    </tr>
+	<tr>
+	  <td>
+		<img
+		  src="https://user-images.githubusercontent.com/32434520/180445548-63d5e0ce-635f-4e7b-bed7-e4bcbf2dc8c4.png"
+		/>
+	  </td>
+	  <td>
+		<img
+		  src="https://user-images.githubusercontent.com/32434520/180447103-76ff1e25-ec35-4e7f-bd54-9d98545ca1df.png"
+		/>
+		<img
+		  src="https://user-images.githubusercontent.com/32434520/180446790-50b6a53b-119e-4b74-a08d-dda146fb9f29.png"
+		/>
+	  </td>
+	</tr>
```

Comment:
Hi, @debjit20504, it seems that the markdown editor you use automatically formatted the markdown code. I hope you can keep the unrelated code as they were before.

Could you make changes and push another commit to this PR?

Reply:
ok, I'll do that

Reply:
[README.md](https://github.com/hypertrons/hypertrons-crx/files/9173238/README.md)
[README.zh-CN.md](https://github.com/hypertrons/hypertrons-crx/files/9173239/README.zh-CN.md)